### PR TITLE
BSRD-629: Move invoke tasks in a separate package

### PR DESCRIPTION
**Pull request by yankinmax**
https://jira.camptocamp.com/browse/BSRD-629

### CODE REVIEW

File: tasks/main.py [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+
+
+class Tasks(Program):
+    def core_args(self):
+        core_args = super().core_args()
+        extra_args = [
+            Argument(names=("argument", "agr"), help="Foo the bars"),
+            # ...
+        ]
+        return core_args + extra_args
```
Comment (yankinmax) [inline tasks/main.py:24]: here we can provide arguments for commands

File: setup.py [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+    url="https://github.com/camptocamp/odoo-project-tasks",
+    packages=["tasks"],
+    entry_points={"console_scripts": ["invoke-cli=tasks.main:program.run"]},
+    include_package_data=True,
+    install_requires=requirements,
+    python_requires=">=3.7",
+    setup_requires=["setuptools_scm"],
+    license="MIT license",
+    zip_safe=False,
+    keywords="invoke-cli",
```
Comment (simahawk) [inline setup.py:45]:
```suggestion
    keywords="odoo-tools-cli",
```

File: HISTORY.rst [hunk 3]
```diff
@@ -0,0 +1,28 @@
+.. :changelog:
```
Comment (simahawk) [inline HISTORY.rst:1]: I would remove this file and let [towncrier](https://pypi.org/project/towncrier/) to the job w/ commit msgs

File: .pre-commit-config.yaml [hunk 4]
```diff
@@ -0,0 +1,44 @@
+files: ^tasks/
```
Comment (simahawk) [inline .pre-commit-config.yaml:1]: please include these changes

https://github.com/camptocamp/odoo-template/pull/646
https://github.com/camptocamp/odoo-template/pull/678

**Review by simahawk — CHANGES_REQUESTED** [review]

**Comment by yankinmax** [comment]
@simahawk I've mostly updated with your suggestions. I don't know how towncrier works or how to check if it's working. I added needed files. It would be next step, I would say.
@gurneyalex I've checked all tasks. Only on few I have questions. See https://jira.camptocamp.com/browse/BSRD-629

I think this PR as the first step could be merged as it's. Then probably we can create separate cards about tasks to fix. Also I pointed to the fact that scripts should be checked after we have new directory structure. This was tested on current structure and we have a lot of code snippets with hardcoded `odoo/local-src`, `odoo/external-src` etc.

**Pull request merged**